### PR TITLE
ERB Examples for SE Email Integration

### LINKED ISSUE #867: Add subject erb example to email integration doc
What the title says.

### PR DESCRIPTION
**Pull request by asachs01**
## Description
Adds erb template examples

## Motivation and Context
Closes #867 

## Review Instructions
Add the examples to your SE deployment and confirm that they work as expected

### CODE REVIEW

File: content/sensu-enterprise/3.3/integrations/email.md [hunk 1]
```diff
@@ -290,7 +293,33 @@ type         | String
 required     | false
 example      | {{< highlight shell >}}"body": "/etc/sensu/email/body_template.erb"{{< /highlight >}}
 
+##### Example Templates
```
Comment (rgeniesse) [inline content/sensu-enterprise/3.3/integrations/email.md]: We already have an example for the body here:

https://docs.sensu.io/sensu-enterprise/3.3/integrations/email/#custom-email-templates-example

Should this addition be removed and the subject addition added to where the above link is?

**Review by rgeniesse — DISMISSED** [review]
See inline comments.

**Comment by asachs01** [comment]
@rgeniesse I've updated the doc per your suggestion. Can you take a gander?

**Comment by rgeniesse** [comment]
@asachs01 The spaces on the tables need to be removed. They cause the tables to get all messed up.

I must have missed the ToC the first time around, but it is not formatting properly with anchors showing up post render:

screen shot 2019-01-04 at 8 50 55 am

**Comment by asachs01** [comment]
@rgeniesse can you check again? Should be good this go around.

File: content/sensu-enterprise/3.3/integrations/email.md [hunk 2]
```diff
@@ -37,11 +40,16 @@ ERB templating syntax][5]._
 
 ### Example(s) {#custom-email-templates-example}
```
Comment (rgeniesse) [inline content/sensu-enterprise/3.3/integrations/email.md]:
```suggestion
### Example(s) {#examples-custom-email-templates-example}
```

File: content/sensu-enterprise/3.3/integrations/email.md [hunk 3]
```diff
@@ -13,13 +13,16 @@ users only.**
 
 - [Overview](#overview)
 - [Custom email templates](#custom-email-templates)
-  - [Example(s)](#custom-email-templates-example)
+  - [Example(s)](#examples-custom-email-templates-example)
 - [Configuration](#configuration)
-  - [Example(s)](#examples)
-  - [Integration Specification](#integration-specification)
+  - [Example(s)](#examples-examples)
```
Comment (rgeniesse) [inline content/sensu-enterprise/3.3/integrations/email.md]:
```suggestion
  - [Example(s)](#examples)
```

File: content/sensu-enterprise/3.3/integrations/email.md [hunk 4]
```diff
@@ -13,13 +13,16 @@ users only.**
...
+  - [Integration specification](#integration-specification)
     - [`email` attributes](#email-attributes)
     - [`smtp` attributes](#smtp-attributes)
+      - [EXAMPLE](#example-smtp-attributes-example)
```
Comment (rgeniesse) [inline content/sensu-enterprise/3.3/integrations/email.md]:
```suggestion
      - [EXAMPLE](#smtp-attributes-example)
```

**Comment by rgeniesse** [comment]
@asachs01 I've added 3 changes, as the links didn't actually go anywhere with the ToC. Took a guess at what they should be. Confirmed each one works at least post change.

File: content/sensu-enterprise/3.3/integrations/email.md (hunk 2 above)
Comment (rgeniesse) [reply content/sensu-enterprise/3.3/integrations/email.md]: There's a space being removed here, not sure why. I only wanted the `examples-` to be added on.

**Comment by asachs01** [comment]
@rgeniesse this should be good if I can get a :+1: on it.

**Review by rgeniesse — APPROVED** [review]
LGTM! 👍

**Pull request merged**